--- a/include/feeds.mk
+++ b/include/feeds.mk
@@ -35,7 +35,7 @@
 # 1: destination file
 define FeedSourcesAppendOPKG
 ( \
-  echo 'src/gz %d_core %U/$(DATE)_$(VERSION_CODE)_$(BRANCH)/targets/%S/packages'; \
+  echo 'src/gz %d_core %U/targets/%S/packages'; \
   $(strip $(if $(CONFIG_PER_FEED_REPO), \
 	echo 'src/gz %d_base $(OWRT_LINK)/packages/%A/base'; \
 	$(if $(CONFIG_BUILDBOT), \
@@ -51,7 +51,7 @@
 ( \
   echo '# This file is auto-generated and build-specific, any changes will be intentionally lost in sysupgrade.'; \
   echo '# Add your custom feeds to /etc/apk/repositories.d/customfeeds.list'; \
-  echo '%U/$(DATE)_$(VERSION_CODE)_$(BRANCH)/targets/%S/packages/packages.adb'; \
+  echo '%U/targets/%S/packages/packages.adb'; \
   $(strip $(if $(CONFIG_PER_FEED_REPO), \
 	echo '$(OWRT_LINK)/%A/base/packages.adb'; \
 	$(if $(CONFIG_BUILDBOT), \
